### PR TITLE
Extend holidays checker

### PR DESCRIPTION
**Pull request by alexey-klimuk**
add holidays gem, extend bank holidays checker by adding Belgium holidays, add specs

### CODE REVIEW

**Comment by alexey-klimuk** [comment]
[[TARIFF17] build into the alert a checker that if it was a bank holiday yesterday/today then to add a day onto the checker](https://trello.com/c/J6t7OQOi/1673-tariff17-build-into-the-alert-a-checker-that-if-it-was-a-bank-holiday-yesterday-today-then-to-add-a-day-onto-the-checker)

File: spec/models/measure_condition_spec.rb [hunk 1]
```diff
@@ -154,4 +154,38 @@
       end
     end
   end
+
+  describe '#document_code' do
```
Comment (matthewford) [inline spec/models/measure_condition_spec.rb]: not sure if this measure condition spec belongs in this commit?

File: lib/bank_holidays.rb [hunk 2]
```diff
@@ -5,13 +5,27 @@ module BankHolidays
...
       Net::HTTP.get(URI(URL))
     end
     dates = JSON.parse(response)['events'].map{ |e| Date.parse(e['date']) }
     dates.select!{ |d| d <= Date.today }
-    dates += ((Date.today - n + 1)..Date.today).to_a.select{ |d| d.saturday? || d.sunday? }
     dates.sort.last(n)
   end
+
+  def self.other(n)
+    Holidays.between(Date.today - n, Date.today, :be_nl, :be_fr, :gb)
```
Comment (matthewford) [inline lib/bank_holidays.rb]: I think bl_nl is enough - bl_fr is just a translations

File: lib/bank_holidays.rb [hunk 3]
```diff
@@ -5,13 +5,27 @@ module BankHolidays
   URL = 'https://www.gov.uk/bank-holidays/england-and-wales.json'
```
Comment (matthewford) [inline lib/bank_holidays.rb]: We can remove the GOV.UK Parsing, if we use the gem for both GB and BE holidays.

File: spec/models/measure_condition_spec.rb (hunk 1 above)
Comment (alexey-klimuk) [inline spec/models/measure_condition_spec.rb]: @matthewford I know that it's not belong, should I remove spec?
just covering uncovered methods

Comment (matthewford) [inline spec/models/measure_condition_spec.rb]: maybe just split into a different pr
